plugins/adobe-analytics-compatible-version/plugin.test.ts

```diff
@@ -144,6 +161,17 @@ describe('Adobe Analytics Compatible Version', () => {
     const result = plugin([configurationEvent]);
 
     expect(result).toMatchObject({
+      events: [],
+      message:  "No version info could be found. Either the Assurance SDK isn't registered or the SDK did not pass in cached events upon activating.",
+      result: 'unknown'
+    });
+  });
+
+  it('should return unknown when no analytics version events are found', () => {
+    const result = plugin([noAnalyticsVersionEvent]);
```

Comment:
nit: this is a bit confusing as here you are referring to the shared state and not a specific version event. Can we rename this to `noAnalyticsVersionSharedState` or simply `noAnalyticsVersion`

`should return unknown when no analytics version is found in hub shared state`